tests/foreman/cli/test_errata.py

```diff
@@ -920,7 +916,12 @@ def cleanup():
     ids=('org_id', 'org_name', 'org_label', 'no_org_filter'),
 )
 def test_positive_list_filter_by_org_sort_by_date(
-    module_sca_manifest_org, rh_repo, custom_repo, filter_by_org, sort_by_date, module_target_sat
+    module_sca_manifest_org,
+    rh_repo_module_manifest,
```

Comment:
Can we remove `rh_repo` fixture as you're replacing it with `rh_repo_module_manifest` and both looks same?

Reply:
sounds good will do for next commit